packages/kiota_abstractions/lib/src/extensions/request_information_extensions.dart

```diff
@@ -137,16 +137,16 @@ extension RequestInformationExtensions on RequestInformation {
     content = writer.getSerializedContent();
   }
 
-  void configure<T>(void Function(RequestConfiguration)? configurator) {
+  void configure<T extends AbstractQueryParameters>(void Function(RequestConfiguration<T>)? configurator, T Function() createParameters) {
```

Comment:
Correct me if I'm wrong, but the `configurator` may be null, but the second parameter can't? IMHO they should both be nullable or neither should.

Reply:
Both params can now be null.